src/installer/Directory.Build.props

```diff
@@ -187,6 +189,13 @@
         <TargetsUnix>true</TargetsUnix>
       </PropertyGroup>
     </When>
+    <When Condition="$(OutputRid.StartsWith('android'))">
```

Comment:
@akoeplinger is <TargetsLinux> required here?

Reply:
probably not required if it's not used but in general the `Targets*` seem to follow the RID graph which has Android->Linux->Unix dependency.